web/components/comments-list.tsx

```diff
@@ -20,12 +16,20 @@ import { LoadingIndicator } from './loading-indicator'
 
 const COMMENTS_PER_PAGE = 50
 
-type ContractComment = Comment & { contractId: string }
+type ContractComment = Comment & {
+  contractId: string
+  contractSlug: string
+  contractQuestion: string
+}
+
+function contractPath(slug: string) {
+  // in honor of austin, who insists that contract URLs are prefixed with a username
+  return `/Austin/${slug}`
+}
```

Comment:
Hmm, we know the comment's `userUsername`, can't we just pass that in? This is just a joke, right?

Reply:
No, this is for the URL to the contract. It wouldn't make any more sense for it to be the commenter's name. It's "supposed" to be the contract creator's name, but it has no effect.

Reply:
If you are a robot who doesn't think it's funny to put "Austin" the most dignified thing would be to put "market" or something.

Reply:
I see, but users might share the URL after and then it would not have the correct info. Referral bonuses are also based on the creator of the contract. 

I don't think our users will get this joke. And why put "market" if everywhere else we use the creator username? I don't get it.

Reply:
> I see, but users might share the URL after and then it would not have the correct info. Referral bonuses are also based on the creator of the contract.

Referral bonuses are based on the creator of the contract. They are not based on this string. This string has no effect.

Are you seriously suggesting I go add this extra database field to put here? If you really want me to, I will, but I think it's a total waste of time.

Reply:
`creatorUsername` is already a comment field! Plus we know it anyway from which user page this is.

Reply:
Oh I see `userUsername` is a field, not `creatorUsername`. But we could still pass in the creator username if we know it.

Reply:
One more idea is that after a contract loads, we could replace the url with the correct username if it wasn't correct.